src/qibolab/platform.py

```diff
@@ -359,6 +373,28 @@ def get_qubit(self, qubit):
         except KeyError:
             return list(self.qubits.keys())[qubit]
 
+    # TODO: Similar for couplers (I think this won't be needed)
+    def get_coupler(self, coupler):
+        """Return the name of the physical coupler corresponding to a logical coupler.
+
+        Temporary fix for the compiler to work for platforms where the couplers
+        are not named as 0, 1, 2, ...
+        """
+        try:
+            return self.couplers[coupler].name
+        except KeyError:
+            return list(self.couplers.keys())[coupler]
+
+    def get_coupler_pair(self, qubit_pair):
+        try:
+            return qubit_pair.coupler.name
+            # for coupler in self.couplers.values():
+            #     # I add this as a subset as it may appear on the future
+            #     if set(qubits).issubset(coupler.qubits):
+            #         return coupler.name
+        except AttributeError:
+            return None
```

Comment:
Are these actually used anywhere, other than the tests (from where we can easily remove)? I don't think so, for the qubit they are needed before we have `create_*` methods for the native gates, but there are no such methods for couplers. Maybe we can remove completely.

Reply:
I suppose they could get involved when the `create_*` calls a 2q gate that needs coupler interaction, right ?

Reply:
I would say no, because the error these functions are capturing appears only because the compiler is calling them with wrong qubit names (basically using the qibo qubit names 0, 1, 2,..., instead of the platform qubit names). I think the compiler will never deal with couplers because they are not logical qubits. So even for 2q gates, they will only appear within the `create_*` not in its arguments and there the platform knows how to handle them.

But anyway, at some point we will fix the compiler and remove the `Platform.get_qubit` methods so at that point we will also remove these if they exist. I just mentioned it because searched the code and could not find any occurence.

Reply:
I can remove them now